### PR TITLE
Feature/codeof conduct

### LINKED ISSUE #1: Create Code of Conduct
Create and upload code of conduct for project github

### PR DESCRIPTION
**Pull request by Terrel-Deligny**
merging code of conduct to master

### CODE REVIEW

File: Code_of_Conduct.md [hunk 1]
```diff
@@ -1 +1,49 @@
-Test - COC to be added later
\ No newline at end of file
+Code of  Conduct
```
Comment (Calixte-Williams) [inline Code_of_Conduct.md]: Add header format to heading / titles

File: Code_of_Conduct.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+
+As software developers; it is important to create and adhere to a Code of Conduct especially when working in a team. A code of conduct is a set of guidelines that specify how a person is expected to behave while associated with an organization. Below specifies the Code of Conduct guidelines for our group while we work together to complete our project.
+
+
+
+
+Code of Conduct guidelines: Self
+
+Will not use language that is generally regarded as having the potential to offend others, whether spoken orally or in writing (including emails, texts, and posts on slack etc). This covers any derogatory remarks about a person's race, gender, or sexual orientation as well as any derogatory remarks about their religion, personal convictions, or right to free speech.
```
Comment (Calixte-Williams) [inline Code_of_Conduct.md]: We will ****
Correct grammar for Code of Conduct

File: Code_of_Conduct.md [hunk 3]
```diff
@@ -1 +1,49 @@
...
+Disrupting or preventing a team member from contributing to project wether it be talking over others who are already speaking or excessively loud talking or shouting disrupting workflow.
```
Comment (Calixte-Williams) [inline Code_of_Conduct.md]: whether*

File: Code_of_Conduct.md [hunk 4]
```diff
@@ -1 +1,49 @@
...
+Perform work only in areas of competence.
+
+
+
+
+
+
+Code of Conduct guidelines: Management
+
+Team Members will put the project first before personal preferences and will do nothing to jeopardize the outcome of this project.
```
Comment (Calixte-Williams) [inline Code_of_Conduct.md]: Use bullet points to clearly differentiate guidelines.

**Review by Calixte-Williams — CHANGES_REQUESTED** [review]

File: Code_of_Conduct.md (hunk 2 above)
Comment (Terrel-Deligny) [reply Code_of_Conduct.md]: Code of Conduct Format fixed

File: Code_of_Conduct.md (hunk 3 above)
Comment (Terrel-Deligny) [reply Code_of_Conduct.md]: Updated

File: Code_of_Conduct.md (hunk 4 above)
Comment (Terrel-Deligny) [reply Code_of_Conduct.md]: Updated

**Review by Calixte-Williams — APPROVED** [review]

**Review by Jermon1 — APPROVED** [review]

**Pull request merged**